### ISSUE TITLE
Using Chrome's RtcQuicTransport as a client?

### ISSUE THREAD

**Issue by cryptix** [lucas-clemente/quic-go#2696]
I recently tried the [RtcQuicTransport](https://github.com/shampson/RTCQuicTransport-Origin-Trial-Documentation) trial experiment in recent chrome versions but couldn't get it to connect properly against my server code using `v0.17.3` of this package.

`Failed to establish a connection to quic-transport://testhost:1234/: net::ERR_QUIC_PROTOCOL_ERROR.QUIC_INVALID_VERSION (Client may support one of the versions in the server's list, but it's going to close the connection anyway. Supported versions: {ff00001b}, peer supported versions: {}).`

The readme here links to [this wiki page about the quic versions](https://github.com/quicwg/base-drafts/wiki/QUIC-Versions) so that hex number means draft-ietf-quic-transport-27, right?

When I try to add `0xff00001b` to `quic.Config.Versions` I get this error on startup of my server code:
`2020/07/31 07:46:34 0xff00001b is not a valid QUIC version`

So I _guess_ it's expected behavior...? I saw on the [working group page](https://github.com/quicwg/base-drafts/wiki/19th-Implementation-Draft) that the current draft is 29. Can I try an older version of this package to get it to work?

ps: I'm more interested in the use of the transport directly than using it through http/3.

**Comment by marten-seemann** [comment]
We don't support draft-27 any more.
As far as I know, Chrome supports draft-29 already (but maybe only in Beta or Canary?). Have you tried that?

**Comment by cryptix** [comment]
Hm.. I just tried canary and ran into the same problem (client only advertises 27).

It might be related to the specific RtcQuicTransport API. I can try http/3 later but that’s not interesting for me since i want to make a POC for bidirectional streams created from both sides and possible WebRTC-like things with ICE, too later.

**Comment by marten-seemann** [comment]
Did you use `--quic-version=h3-29`?

**Comment by cryptix** [comment]
Yup, and it seems like that doesn't influence the pure `QuicTransport` trial API. The HTTP/3 example works but not the direct use of transport.

I just found [this thread on the WebTransport ML](https://groups.google.com/a/chromium.org/forum/#!topic/web-transport-dev/beNgT5xnLDA) and it looks like draft-29 was added around June 16th.

I will see how hard it is to build a nightly and otherwise wait for the next canary.
